packages/shared/src/types/targeting/header.ts

```diff
@@ -2,6 +2,11 @@ import { PageTracking } from './shared';
 
 export type Edition = 'UK' | 'US' | 'AU' | 'INT';
 
+export interface purchaseInfo {
```

Comment:
I don't fully understand the requirements here, do these fields essentially override the existing `userCohort` field?
Is there an explanation of the `userType` values somewhere, and how dotcom determines them?

Reply:
I actually don't fully understand the values either. I know we need to show a different CTA based on whether a user needs to register or just sign in. Not sure what the distinction between `new` and `guest` is, will get in touch with the conversions team for more details. But in any case, this info will be plucked from a JSON encoded cookie named `GU_CO_COMPLETE`. See guardian/dotcom-rendering/pull/4857 for the draft implementation. The cookie does not exist in CODE / prod environments yet so I cannot verify that all the parts mesh together correctly, but this is the spec I have been given.

Reply:
👍 
A couple of small things:
1. the convention is to capitalise types, so `PurchaseInfo`
2. the fields do not need to be optional (they're not in the cookie [model](https://github.com/guardian/support-frontend/pull/3743/files#diff-83d8ad95a6e5a65c9f8c11c0c329bebfd9716bc691c9312a3b62e3f43ff61332R150))